website/docs/components/stepper/nav/partials/specifications/states.md

```diff
@@ -0,0 +1,3 @@
+## States
+
+![Stepper Nav states](/assets/components/stepper/nav/stepper-nav-states.png =477x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=477x*"